analysis/aws_rules_s3_access_logs/aws_s3_access_error.yml

```diff
@@ -21,7 +21,7 @@ Tests:
     ExpectedResult: false
     Resource: 
       {
-        "errorCode": "-"
```

Comment:
Shouldn't we add a new test case that looks at `otherFields`?

Reply:
Not sure I understand, I just made up the name `otherFields` so it wouldn't be an empty dictionary (not that it can't be an empty dictionary). What would you like the test to look like?

Reply:
I missed the context on the removal of `-` in the logs, disregard!